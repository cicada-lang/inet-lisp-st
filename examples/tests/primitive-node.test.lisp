(define (main)
  (= x x-op (link))
  (= y y-op (link))
  (= z (iadd x y))
  (connect x-op 1)
  (connect y-op 2)
  z)

(main)
