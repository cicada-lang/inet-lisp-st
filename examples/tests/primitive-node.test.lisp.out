<net>
:root -<[3]
</net>

